workspace/solution.go

```diff
@@ -67,11 +70,13 @@ func (s *Solution) Write(dir string) error {
 		return err
 	}
 
-	path := filepath.Join(dir, solutionFilename)
+	path := filepath.Join(dir, ignoreSubdir)
```

Comment:
Just a heads up if we go with one const for filename path then we have to use `filepath.Dir(SolutionFilename)` to just get the directory path and create that if it doesn’t already exist